Objects/rangeobject.c

```diff
@@ -182,6 +204,12 @@ compute_range_length(PyObject *start, PyObject *stop, PyObject *step)
     Algorithm is equal to that of get_len_of_range(), but it operates
     on PyObjects (which are assumed to be PyLong objects).
     ---------------------------------------------------------------*/
+
+    // fast path when all arguments fit into a long integer
```

Comment:
Let's keep the existing style in this function where variable declarations come before uses. (It looks odd to have a big block of declarations that _isn't_ at the top of the function.)